src/adapters/refract/transformTransactions.coffee

```diff
@@ -65,11 +65,12 @@ module.exports = (transactions, options) ->
 
     # Check for empty http request
     requestName = _.chain(httpRequest).get('meta.title', '').contentOrValue().value()
-    requestHeaders = getHeaders(httpRequest)
     requestBody = trimLastNewline(if _.content(httpRequestBody) then _.content(httpRequestBody) else '')
     requestSchema = trimLastNewline(if _.content(httpRequestBodySchemas) then _.content(httpRequestBodySchemas) else '')
     requestAuthSchemes = transformAuth(httpTransaction, options)
 
+    [requestHeaders, requestHeaders1A] = getHeaders(httpRequest)
```

Comment:
I agree that returning arrays with two (or more) different data structures is not very elegant and good for readibility, but I wanted to keep it DRY. I will add `getHeaders1A`.